### DISCUSSION TITLE
Error occurs when import ase calculators of LAMMPSlib and DP

### DISCUSSION THREAD

**Discussion by phys-chem** [deepmodeling/deepmd-kit#2869] [Q&A]
I meet this error: 
[libprotobuf ERROR google/protobuf/descriptor_database.cc:642] File already exists in database: tensorflow/compiler/xla/xla_data.proto
[libprotobuf FATAL google/protobuf/descriptor.cc:1984] CHECK failed: GeneratedDatabase()->Add(encoded_file_descriptor, size):

when importing DP with LAMMPSlib at the same time:
```
from deepmd.calculator import DP
from ase.calculators.lammpslib import LAMMPSlib
```
```
if i didn't import DP, no errors occur. Here is a test program:

from ase import Atom, Atoms
from ase.calculators.lammpslib import LAMMPSlib
from ase.optimize import BFGS,FIRE
from ase.constraints import ExpCellFilter
from deepmd.calculator import DP

cmds = ["pair_style lj/cut 3.0", "pair_coeff * * 1 1" ]
lat = [3.1, 3.1, 3.15, 90, 90, 95]
alloy = Atoms( [Atom('Mg', [0, 1.8, 1.3]), Atom('Mg', [1.6, 0.9, 2.9])] , cell = lat, pbc=True  )

lammps = LAMMPSlib(lmpcmds=cmds, logfile='test.log', keep_alive = True )
alloy.calc = lammps

sys = ExpCellFilter(alloy)
dyn = BFGS(atoms= sys, trajectory=  'test.traj')
dyn.run(fmax=0.001, steps=50)
```

**Comment by njzjz** [comment]
The reason may be that two different protobuf libraries are loaded (when the Python and C++ interface are compiled against different protobuf) and protobuf doesn't allow this behavior.

**Reply by phys-chem** [reply]
Thanks for the reply! Do you know some methods to solve it?
